heft-plugins/heft-webpack4-plugin/src/index.ts

```diff
@@ -1,18 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import type { IHeftPlugin } from '@rushstack/heft';
+export { PLUGIN_NAME as PluginName } from './WebpackPlugin';
 
-import { WebpackPlugin } from './WebpackPlugin';
-
-export {
+export type {
   IWebpackConfigurationWithDevServer,
   IWebpackConfiguration,
-  IWebpackBuildStageProperties,
-  IWebpackBundleSubstageProperties
+  IWebpackPluginAccessor,
+  IWebpackVersions
 } from './shared';
```

Comment:
Are these projects not running API Extractor? They should be.

Reply:
They are. I just haven't included the md updates in these PRs due to the non-building nature of this branch. They'll be in the larger PR made into main